sentry-android-integration-tests/sentry-uitest-android-benchmark/build.gradle.kts

```
@@ -101,12 +101,12 @@ dependencies {
 
     androidTestImplementation(projects.sentryTestSupport)
     androidTestImplementation(libs.kotlin.test.junit)
-    androidTestImplementation(Config.TestLibs.espressoCore)
-    androidTestImplementation(Config.TestLibs.androidxTestCoreKtx)
-    androidTestImplementation(Config.TestLibs.androidxRunner)
-    androidTestImplementation(Config.TestLibs.androidxTestRules)
-    androidTestImplementation(Config.TestLibs.androidxJunit)
-    androidTestUtil(Config.TestLibs.androidxTestOrchestrator)
+    androidTestImplementation(libs.androidx.test.espresso.core)
+    androidTestImplementation(libs.androidx.test.core.ktx)
+    androidTestImplementation(libs.androidx.test.ext.junit)
+    androidTestImplementation(libs.androidx.test.orchestrator)
```

```suggestion
    androidTestUtil(libs.androidx.test.orchestrator)
```
I think this is what is causing the build failure